explorer/query_hub.go

```diff
@@ -245,7 +245,12 @@ func (s *LocalServices) cacheUpstreamQueryPackBundle(ctx context.Context, mrn st
 		return nil, errors.New("failed to retrieve upstream query pack " + mrn + ": " + err.Error())
 	}
 
-	bundleMap := bundle.ToMap()
+	bundleMap, err := bundle.Compile(ctx)
```

Comment:
This is what we also do with default packs in case no upstream is configured.

But for cnspec, we do this server-side when loading the policies. So, is this the correct place?

Reply:
cnquery uses `GetBundle` in its datalake, which should likely be renamed to `GetValidatedBundle` to align with cnspec. That said, it `retrieves and if necessary updates the pack.`. From this definition alone, this change requires the bundle to be returned in a fully computed state - including computed filters. 

Thus we don't want to compile the bundle here. It must be returned in a compiled state.

One thing to do would be to migrate over a few of the tests that are missing to validate this behavior (which I missed when I originally split these methods out).